### PR TITLE
BSIP60: BitShares URI Scheme

### PR DESCRIPTION
**Pull request by jhtitor**
Discussion: https://github.com/bitshares/bsips/issues/131

### CODE REVIEW

**Comment by pmconrad** [comment]
@ryanRfox please assign a number

File: bsip-00xx.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+It would win us some bytes in the URL, although it would also make parsers a bit harder to write. The proposed short versions are:
+
+|full type   | short type|
+|------------|-----------|
+|object      | ob        |
+|operation   | op        |
+|block       | bl        |
+|transaction | trx       |
+
+In addition, instead of spelling out operation type (e.g. "transfer"), it's id number could be used (so the `operation/transfer?to=...` could be shortened to `op/0?to=...`).
```
Comment (pmconrad) [inline bsip-00xx.md]: Typo: it's

File: bsip-00xx.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
+    BSIP: <TBD>
```
Comment (ryanRfox) [inline bsip-00xx.md]: Please rename file to "bsip-0060.md"

File: bsip-00xx.md [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+blind_receipt_path   = "blind_receipt" "/" base58_blind_receipt
+base58_blind_receipt = base58 *base58
+base58               = "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" |
+                       "9" | "A" | "B" | "C" | "D" | "E" | "F" | "G" |
+                       "H" | "J" | "K" | "L" | "M" | "N" | "P" | "Q" |
+                       "S" | "T" | "U" | "V" | "W" | "X" | "Y" | "Z" |
+                       "a" | "b" | "c" | "d" | "e" | "f" | "g" | "h" |
+                       "i" | "j" | "k" | "m" | "n" | "o" | "p" | "q" |
+                       "r" | "s" | "t" | "u" | "v" | "w" | "x" | "y" |
+                       "z"
```
[the remark this answers is not in the document]
Comment (jhtitor) [reply bsip-00xx.md]: woah, great catch

File: bsip-00xx.md (hunk 1 above)
Comment (jhtitor) [reply bsip-00xx.md]: it's embarrassing...

File: bsip-0060.md [hunk 4]
```diff
@@ -0,0 +1,338 @@
+    BSIP: 0060
+    Title: BitShares URI scheme
+    Authors: John Titor, sschiessl, abitmore
```
Comment (ryanRfox) [inline bsip-0060.md]: Sorry, I missed this earlier: Please change Authors to "John Titor, Stefan Schießl, Abit More" to match the README.md file. Thanks

**Comment by pmconrad** [comment]
@ryanRfox please check and merge if ok

**Pull request merged**